source/ocsp-support/ocsp-support.rst

```diff
@@ -181,32 +181,48 @@ response.
 MongoClient Configuration
 --------------------------
 
-This specification introduces the client-level configuration option
-defined below. Drivers that can, on a per MongoClient basis, disable
-non-stapled OCSP while keeping stapled OCSP enabled MUST implement
-this option.
+This specification introduces the client-level configuration options
+defined below.
 
 tlsDisableOCSPEndpointCheck
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This boolean option determines whether MongoClient should refrain from
+Drivers that can, on a per MongoClient basis, disable non-stapled OCSP
+while keeping stapled OCSP enabled MUST implement this option.
+
+This boolean option determines whether a MongoClient should refrain from
 reaching out to an OCSP endpoint i.e.  whether non-stapled OCSP should
 be disabled.  When set to true, a driver MUST NOT reach out to OCSP
 endpoints. When set to false, a driver MUST reach out to xOCSP
 endpoints if needed (as described in :ref:`Suggested OCSP Behavior`).
 This option MUST default to false.
 
+tlsDisableCertificateRevocationCheck
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Drivers that support an option to toggle general certificate
```

Comment:
It seems like this could be interpreted to mean "if you can support this option, then you must implement it".

Given the previous conversation, and the description of "Backwards Compatibility", maybe this should be something like: "If enabling revocation checking in your TLS library considers certificates with no revocation mechanisms a hard-failure (i.e. no CRL distribution points, OCSP stapling, or OCSP endpoints), then implement this option".

Reply:
Hm, I could see how that could be misinterpreted true.
How's this?
"Drivers whose TLS libraries support an option to toggle general certificate revocation checking must implement this option if enabling general certificate revocation checking causes hard-fail behavior when no revocation mechanisms are available (i.e.. no methods are defined or the CRL distribution points/OCSP endpoints are unreachable)."

Reply:
SGTM!